### ISSUE TITLE
Color not returned for text (wrong variable being set?)

### ISSUE THREAD

**Issue by uprichard** [pdfminer/pdfminer.six#184]
In pdfinterp.py the color is put into self.graphicstate.color but I think it should be self.graphicstate.ncolor 

For example:
```
    # setrgb-non-stroking
    def do_rg(self, r, g, b):
        self.graphicstate.color = (r, g, b)
        #self.do_cs(LITERAL_DEVICE_RGB)
        return
```

Should be
```
    # setrgb-non-stroking
    def do_rg(self, r, g, b):
        self.graphicstate.ncolor = (r, g, b)
        #self.do_cs(LITERAL_DEVICE_RGB)
        return
```

**Comment by hagen666** [comment]
Is there any places calling the function?

**Comment by OisinMoran** [comment]
This was fixed with PR https://github.com/pdfminer/pdfminer.six/pull/129.

See https://github.com/pdfminer/pdfminer.six/commit/2231f0892e0dd0a9af0d69ed73dc4bc8b2823245#diff-2275db414f4c92f9920df88b9b22446d

**Comment by pietermarsman** [comment]
Fixed in #287 and #298.
